s3.tf

```diff
@@ -1,7 +1,41 @@
 resource "aws_s3_bucket" "demo-example" {
```

Comment:
Accurics remediation:
Title: HIGH: S3 Buckets Server Side Encryption At Rest
Description:Vulnerability: Ensure that S3 Buckets have server side encryption at rest enabled to protect sensitive data./n/nRemediation:On the AWS S3 console, for each bucket that failed the rule, navigate to the Properties tab, and select Default encryption. Select an encryption method (AES-256 or AWS-KMS). Refer to: https://docs.aws.amazon.com/AmazonS3/latest/user-guide/default-bucket-encryption.html